frontend/front/src/pages/Surveyor/map/MapLink.js

```diff
@@ -28,7 +28,7 @@ function openGoogleMaps(locations) {
 const MapLink = ({ homeData }) => {
   const [dashboardHomeData, setDashboardHomeData] = useState(homeData);
 
-  console.log(dashboardHomeData);
+  console.log(dashboardHomeData,"= dashboardHomeData");
```

Comment:
Probably just delete this `console.log`, we don't want these in the codebase long term